code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -227,6 +227,71 @@
 		/obj/item/clothing/gloves/combat, // 0 TC
 		/obj/item/clothing/shoes/combat) // 0 TC
 
+// 175 TC
+/obj/item/storage/box/syndie_kit/bundle/operative
+	name = "\"Operative\" Bundle"
+	desc = "Glory to the Syndicate! Only the essentials for destroying Nanotrasen in this important kit."
+	items = list(
+		/obj/item/mod/control/pre_equipped/traitor, // 30TC
+		/obj/item/card/id/syndi_scan_only, // ~2TC?
+		/obj/item/encryptionkey/syndicate, // 10tc
+		/obj/item/melee/energy/sword/saber/red, // 40tc
+		/obj/item/shield/energy, // 40tc
+		/obj/item/pinpointer/advpinpointer, // 20tc, get dat fuckin disk
+		/obj/item/storage/belt/military, // 10tc
+		/obj/item/grenade/plastic/c4, // 5tc
+		/obj/item/bio_chip_implanter/proto_adrenalin, // 18tc
+		/obj/item/toy/figure/crew/syndie, // 0tc
+		/obj/item/clothing/under/syndicate // 0tc
+	)
+
+// 250 TC worth of credits
+/obj/item/storage/box/syndie_kit/bundle/rich
+	name = "Big Spender Bundle"
+	desc = "It's money. I don't need to explain more."
+	items = list(
+		/obj/item/clothing/under/suit/really_black, // 0 TC
+		/obj/item/clothing/shoes/laceup, // 0 TC
+		/obj/item/clothing/glasses/monocle, // 0 TC
+		/obj/item/clothing/gloves/color/white, // 0 TC
+		/obj/item/clothing/head/that, // 0 TC
+		/obj/item/storage/secure/briefcase, // 0 TC
+		/// syndie briefcase has 600 credits for 5 TC.
+		/obj/item/stack/spacecash/c10000,
+		/obj/item/stack/spacecash/c10000,
+		/obj/item/stack/spacecash/c10000
```

Comment:
that is.... almost double the amount of money available at round start in the entire game round. If security or command got their hands on that they could buy almost 100 shotguns and well over 100 of any laser weaponry from cargo.